Readability.js

```diff
@@ -1722,6 +1722,23 @@ Readability.prototype = {
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
 
+    // Get <div> nodes which have <br> node(s) and append them into the `nodes` variable.
+    // Some articles' DOM structures might look like
+    // <div>
+    //   Sentences<br>
+    //   <br>
+    //   Sentences<br>
+    // </div>
+    var brNodes = this._getAllNodesWithTag(this._doc, ["div > br"]);
+    if (brNodes.length > 1) {
+      var set = new Set();
+      [].forEach.call(brNodes, function(node) {
+        var div = node.parentNode;
+        !set.has(div) && set.add(div);
+      });
+      nodes = [].concat.call(Array.from(set), nodes);
+    }
+
```

Comment:
Hi, @gijsk 

Right, it's really hard to implement a light-weight algorithm for detecting `<div>` nodes with the pattern. But let's try to figure out that.

How about this? We select all `<div>` nodes which have `<br>` nodes. Then try to score them in our original score algorithm.

What do you think? Maybe you have better ideas. :)